File: 2_method.py
class Students:
    def __init__(self, name, roll, group):
        self.name = name
        self.roll = roll
        self.group = group
    
    def display_info(self):
        print(f'Name is {self.name} \nRoll is : {self.roll} \nGroup is : {self.group} \n')
        
    def update_info(self, name, roll, group):
        self.name = name
        self.roll = roll
        self.group = group
        

std = Students('Meraz',6546,'A')
std.display_info()

std.update_info('rakib',232,'B')
std.display_info()

File: 2_method.cpp


#include <bits/stdc++.h>
using namespace std;
#define ll long long
class Student
{
public:
    string name;
    int roll;
    string group;
    Student(string name, int roll, string group)
    {
        this->name = name;
        this->roll = roll;
        this->group = group;
    }

    void display()
    {
        cout << "Name: " << name << endl;
        cout << "Roll: " << roll << endl;
        cout << "Group: " << group << endl;
    }
};
int main()
{
    Student stds("meraz", 232, "A");

    stds.display();

    return 0;
}